src/fmt.ml

```diff
@@ -65,6 +70,10 @@ let rec list ?(pp_sep = cut) pp_v ppf = function
 | v :: vs ->
     pp_v ppf v; if vs <> [] then (pp_sep ppf (); list ~pp_sep pp_v ppf vs)
 
+let hashtbl ?(pp_sep = cut) ~pp_k ~pp_v ppf tbl =
+  let f ppf k v = pf ppf "@[<1>%a:@.%a@]%a" pp_k k pp_v v pp_sep () in
+  Hashtbl.iter (f ppf) tbl
```

Comment:
By similarity with the list function the separator is not fixed and no enclosing is done at all. I considered making the binding separator parametrized too.

But no, I think the enclosing should be left to the user. It's easier to add than to remove enclosing.